java-extensions/hadoop-ext/pom.xml

```diff
@@ -44,6 +44,12 @@
             <version>${iceberg.version}</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.esotericsoftware</groupId>
```

Comment:
why you add dependency to here?

Reply:
It makes the CI failed

Reply:
UnmodifiableCollectionsSerializer needs Kyro dependency.

Reply:
Does this module have any others dependency conflicts?